lib/ansible/modules/network/ovs/openvswitch_bond.py

```diff
@@ -0,0 +1,373 @@
+                for k, v in iteritems(want['external_ids']):
+                    templatized_command = ("%(ovs-vsctl)s -t %(timeout)s"
+                                           " set port %(port)s external_ids:")
+                    command = templatized_command % module.params
+                    command += k + "=" + v
+                    commands.append(command)
+
+    return commands
+
+
+def map_config_to_obj(module):
+    templatized_command = "%(ovs-vsctl)s -t %(timeout)s list-ports %(bridge)s"
+    command = templatized_command % module.params
+    rc, out, err = module.run_command(command, check_rc=True)
+    if rc != 0:
+        module.fail_json(msg=err)
+
+    obj = {}
+
+    if module.params['port'] in out.splitlines():
+        obj['bridge'] = module.params['bridge']
+        obj['port'] = module.params['port']
+
+        templatized_command = ("%(ovs-vsctl)s -t %(timeout)s get"
+                               " Port %(port)s other_config")
+        command = templatized_command % module.params
+        rc, out, err = module.run_command(command, check_rc=True)
+        obj['other_config'] = _other_config_to_dict(out)
+
+        templatized_command = ("%(ovs-vsctl)s -t %(timeout)s get"
+                               " Port %(port)s external_ids")
+        command = templatized_command % module.params
+        rc, out, err = module.run_command(command, check_rc=True)
+        obj['external_ids'] = _external_ids_to_dict(out)
+
+    return obj
+
+
+def map_params_to_obj(module):
+    obj = {
+        'bridge': module.params['bridge'],
+        'port': module.params['port'],
+        'interfaces': module.params['interfaces'],
+        'bond_mode': module.params['bond_mode'],
+        'lacp': module.params['lacp'],
+        'bond_updelay': module.params['bond_updelay'],
+        'bond_downdelay': module.params['bond_downdelay'],
+        'external_ids': module.params['external_ids'],
+        'other_config': module.params['other_config'],
+        'set': module.params['set']
+    }
+
+    return obj
+
+
+def main():
+    """ Entry point. """
+    argument_spec = {
+        'bridge': {'required': True},
+        'port': {'required': True},
+        'interfaces': {'type': 'list'},
+        'bond_mode': {'default': None,
+                      'choices': ['active-backup', 'balance-tcp', 'balance-slb']},
+        'lacp': {'default': None, 'choices': ['active', 'passive', 'off']},
+        'bond_updelay': {'default': None, 'type': 'int'},
+        'bond_downdelay': {'default': None, 'type': 'int'},
+        'state': {'default': 'present', 'choices': ['present', 'absent']},
+        'timeout': {'default': 5, 'type': 'int'},
+        'external_ids': {'default': None, 'type': 'dict'},
+        'other_config': {'default': None, 'type': 'dict'},
+        'set': {'required': False, 'type': 'list', 'default': None}
+    }
+
+    module = AnsibleModule(argument_spec=argument_spec,
+                           supports_check_mode=True)
```

Comment:
Check mode is preferred (I'm implementing it in my modules now, should have from day 1). If you don't support it, change this to `False`.